static/app/components/events/viewHierarchy/wireframe.tsx

```diff
@@ -45,7 +47,10 @@ function Wireframe({hierarchy, selectedNode, onNodeSelect}: WireframeProps) {
 
   const canvasSize = useResizeCanvasObserver(canvases);
 
-  const hierarchyData = useMemo(() => getHierarchyDimensions(hierarchy), [hierarchy]);
+  const hierarchyData = useMemo(
+    () => getHierarchyDimensions(hierarchy, project.platform === 'flutter'),
```

Comment:
this is more of a question but do you know if `"dart-flutter"` should also be included in this change (idk if dart-flutter uses absolute positions and couldnt find a test project)? I just saw it listed as one of the platform categories

Reply:
https://github.com/getsentry/sentry/pull/44582/commits/278e445023e269e8f01648dbac69682649f19a41

Thanks for that, added it and did an `.includes()` check now